src/annotator/sidebar.coffee

```diff
@@ -234,6 +243,7 @@ module.exports = class Sidebar extends Host
       @toolbar.find('[name=sidebar-toggle]')
```

Comment:
Ah, I see - there are already instances of the sidebar manipulating the toolbar's internals directly. That's unfortunate, we shouldn't add any more of that though.